runtimes/node-18.0/server.js

```diff
@@ -30,13 +30,35 @@ const server = micro(async (req, res) => {
         headers[header.toLowerCase()] = req.headers[header];
     });
 
+    const scheme = (req.headers['x-forwarded-proto'] ?? 'http');
+    const host = req.headers['host'].includes(':') ? req.headers['host'].split(':')[0] : req.headers['host'];
+    const port = +(req.headers['host'].includes(':') ? req.headers['host'].split(':')[1] : '80');
+    const path = req.url.includes('?') ? req.url.split('?')[0] : req.url;
+    const queryString = req.url.includes('?') ? req.url.split('?')[1] : '';
+    const query = {};
+    for(const param of queryString.split('&')) {
+        const [ key, value ] = param.split('=');
```

Comment:
This will drop any part of a query param value containing an `=`, which is valid. E.g. `id=1234=abcd` will parse as `{ id: "1234"}`

Reply:
Interesting.. Same for `&` I guess, I will investigate if those symbols are allowed and what is expected behaviour

Reply:
Updated, added new test